internal/pkg/validator/validator.go

```diff
@@ -56,19 +74,18 @@ func Validate(value interface{}, rules ...Validation) error {
 	return nil
 }
 
-func processValidateError(err validator.ValidationErrors) error {
+func processValidateError(err validator.ValidationErrors, translator ut.Translator, fieldName string) error {
 	result := utils.NewMultiError()
 	for _, e := range err {
-		// Remove struct name, first part
-		namespace := regexpcache.MustCompile(`^([^.]+\.)?(.*)$`).ReplaceAllString(e.Namespace(), `$2`)
```

Comment:
Není mi úplně jasný jak se ten field name dostane do err.Namespace(), každopádně při validaci input Rules v něm nic není, tak sem odtamtud předávám fieldName ručně a teprve pokud .Namespace() není prázdný, tak se použije. Je to ok?